store/src/java/com/zimbra/cs/mailbox/calendar/Invite.java

```diff
@@ -511,6 +511,11 @@ public static Invite decodeMetadata(int mailboxId, Metadata meta, CalendarItem c
         String desc = descInMeta ? meta.get(FN_DESC, null) : null;
         String descHtml = descInMeta ? meta.get(FN_DESC_HTML, null) : null;
 
+        // if desc html is missing but desc is present
+        if (desc != null && descHtml == null) {
```

Comment:
Check descHtml first then go for desc..Else it could cause override of descHtml from desc.